### PR TITLE
feat: Added Live-Streaming Functionality

### PR DESCRIPTION
**Pull request by Jish261203**
# What does the PR do? 

Add the  Live Streaming is added to Instructor using ZEGOCLOUD service

*Features* #56

## Type of change ☑️

What sort of change have you made:

1.Made account on ZEGOCLOUD service .
2.selected the specific intergration that is live-streaming.
3.Created two file Room.jsx and Stream.jsx under pages folder

## Screenshot of the changes made in my work:

https://github.com/ashutoshgithubs/iemhacks_DEV-G/assets/127505090/bfe70f8c-72bf-4fb6-9b78-91fc37049d0e

### CODE REVIEW

File: src/pages/Stream.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { ZegoUIKitPrebuilt } from "@zegocloud/zego-uikit-prebuilt";
+
+const Stream = () => {
+  const { streamId } = useParams();
+  
+  function randomID(len) {
+		let result = "";
+		if (result) return result;
+		var chars =
+				"12345qwertyuiopasdfgh67890jklmnbvcxzMNBVCZXASDQWERTYHGFUIOLKJP",
```
Comment (ashutoshgithubs) [inline src/pages/Stream.jsx:12]: Hi @Jish261203 What is this string here under chars. If it has to be protected then you should keep it in env file?? Explain a bit more about this 2 line.

**Review by ashutoshgithubs — COMMENTED** [review]
Hi @Jish261203 Thanks for raising PR.  But as it contains multiple file change, so needs more attention to review this PR. Reviewed some part. I think due to indentation problem at your end, some lines of code came under red colored patch unnecessarily. Can you rectify at your end once and recheck....It will be easier for PA/mentor to go through the review work.

File: src/pages/Stream.jsx (hunk 1 above)
Comment (Jish261203) [reply src/pages/Stream.jsx:12]: Hi @ashutoshgithubs  the string "12345qwertyuiopasdfgh67890jklmnbvcxzMNBVCZXASDQWERTYHGFUIOLKJP" contains a mix of digits (0-9), lowercase letters (a-z), and uppercase letters (A-Z).This string act as pool of characters from ehich function will randomly selects  to create a unique Id

**Comment by ashutoshgithubs** [comment]
Hi @Jish261203 Your PR causing conflict !!

**Pull request closed without merging**